ckanext/visualize/templates/snippets/visualize_view_show.html

```diff
@@ -0,0 +1,78 @@
+{# Adds 'Visualize View' #}
+
+{% resource 'visualize/css/main.css' %}
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+    <title>Toronto CKAN Data Viewer</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css"
```

Comment:
We are not using CKAN's Bootstrap here... we are including it from the BootstrapCDN. Please fix.